Composer/packages/server/src/services/project.ts

```diff
@@ -562,6 +562,10 @@ export class BotProjectService {
         throw new Error('Could not find root bot');
```

Comment:
what about here? (asking because eI have no knowledge)

Reply:
yeah, if the root bot can't be found at the end of the creation process, something bad has happened.